ykrt/src/mt.rs

```diff
@@ -248,7 +248,7 @@ impl MTThread {
             let lp = pack.load(Ordering::Acquire);
             match lp & PHASE_TAG {
                 PHASE_COUNTING => {
-                    let count = lp & !PHASE_TAG;
+                    let count = (lp & !PHASE_TAG) >> 2;
```

Comment:
`2` should be a constant referenced at the top (e.g. `PHASE_NUM_BITS` or whatever).